modular_splurt/code/modules/mob/living/carbon/human/life.dm

```diff
@@ -0,0 +1,4 @@
+/mob/living/carbon/human/BiologicalLife(delta_time, times_fired)
+	..()
+	if(stat != DEAD)
+		update_arousal()
```

Comment:
It'd be better to update the hud whenever the procs that edit the lust amount are called rather than with every tick to minimize lag